sdcm/send_email.py

```diff
@@ -543,10 +543,14 @@ class SlaPerUserEmailReporter(LongevityEmailReporter):
     email_template_file = "results_sl_workloads.html"
 
 
+class ClusterConfigurationTests(BaseEmailReporter):
```

Comment:
Shouldn't it have Reporter in it's name to avoid confusion with the actual test class?